src/pages/overview/releasenotes.md

```diff
@@ -173,6 +173,18 @@ Upgrading to the latest SDK should not break existing applications.
 
 ## Change history
 
+### 4.0.0 (January, 2024; major release)
+| Change  | Language | Description                                                                                   |
+|---------|----------| --------------------------------------------------------------------------------------------- |
+| Changed | Java     | Released SDK v4.0.0, introducing new interfaces fully leveraging power of new PDF Services rest APIs.|
+| Changed | Java     | Updated PDF Properties operation to return PDFProperties object along with JSON string. |
+| Changed | Java     | Updated Extract PDF operation to return content and resourse asset along with content JSON object.|
+| New     | Java     | Added Support for delete asset.|
+| New     | Java     | Added Support for refresh download URI.|
+| New     | Java     | Added Support for callback notifiers for effiecient polling.|
```

Comment:
Can we club all the Added support features.

Reply:
Done